lib/mayaUsd/ufe/UsdUndoMaterialCommands.cpp

```diff
@@ -112,6 +112,74 @@ bool _BindMaterialCompatiblePrim(const UsdPrim& usdPrim)
     return false;
 }
 
+//! Returns true if \p item is a materials scope.
+bool isMaterialsScope(const Ufe::SceneItem::Ptr& item)
```

Comment:
Factored out from `UsdUndoAddNewMaterialCommand::CompatiblePrim` so that it can be reused by other commands in the future.